pex/pep425.py

```diff
@@ -99,6 +99,14 @@ def _iter_supported_tags(cls, impl, version, platform):
     """
     # Predict soabi for reasonable interpreters.  This is technically wrong but essentially right.
     abis = []
+    if impl == 'cp' and version.startswith('2'):
```

Comment:
err, I spoke too soon - looks like the current implementation is:

https://github.com/pypa/pip/blob/develop/pip/pep425tags.py#L85

which seems to have special case for 'u' requiring python < 3.3